test/tests/functional/pbs_reservations.py

```diff
@@ -2311,3 +2314,43 @@ def test_resv_reconfirm_holding_partial_nodes(self):
         solution = '(' + vn[0] + ':ncpus=2)+(' + resv_node2 + ':ncpus=1)'
         a = {'reserve_substate': '5', 'resv_nodes': solution}
         self.server.expect(RESV, a, id=rid)
+
+    def test_standing_resv_with_start_in_past(self):
+        """
+        Test that PBS accepts standing reservations with its start time in the
+        past and end time in future. Check that PBS treats this kind of
+        reservation as a reservation for the next day.
+        """
+
+        now = int(time.time())
+
+        # we cannot use self.server.submit to submit the reservation
+        # because we don't want to specify date in start and end options
+        start = [" -R " + time.strftime('%H%M', time.localtime(now - 3600))]
+        end = [" -E " + time.strftime('%H%M', time.localtime(now + 3600))]
+
+        runcmd = [os.path.join(self.server.pbs_conf['PBS_EXEC'], 'bin',
+                               'pbs_rsub')]
+        tz = ['PBS_TZID=' + self.get_tz()]
+        rule = ["-r \"freq=WEEKLY;BYDAY=SU;COUNT=3\""]
```

Comment:
you can use single quotes for the python string so you don't have to escape the double quotes